### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by poegod** [V4NSH4J/discord-mass-DM-GO#594]
Is it possible to dm a member immediately they join a server?

**Comment by cChimney** [comment]
Yes it is but it's not included with this mass dmer. However you can enable "DM on react" which will dm a person as soon as he reacts, this would work for servers with react verification.
